x-pack/test/apm_api_integration/tests/correlations/failed_transactions.ts

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { registry } from '../../common/registry';
+import { PartialSearchRequest } from '../../../../plugins/apm/server/lib/search_strategies/correlations/search_strategy';
+import { parseBfetchResponse } from '../../common/utils/parse_b_fetch';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const retry = getService('retry');
+  const supertest = getService('supertest');
+
+  const getRequestBody = () => {
+    const partialSearchRequest: PartialSearchRequest = {
+      params: {
+        environment: 'ENVIRONMENT_ALL',
+        start: '2020',
+        end: '2021',
+        kuery: '',
+      },
+    };
+
+    return {
+      batch: [
+        {
+          request: partialSearchRequest,
+          options: { strategy: 'apmFailedTransactionsCorrelationsSearchStrategy' },
+        },
+      ],
+    };
+  };
+
+  registry.when('quynh on trial license without data', { config: 'trial', archives: [] }, () => {
```

Comment:
Just testing stuff out with your own name I assume :D 

```suggestion
  registry.when('on trial license without data', { config: 'trial', archives: [] }, () => {
```

Reply:
Fixed in 909e203 😅